perian/perian-core/src/utils/plugin.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import type { Plugin } from '../../types/types.js'
+import { addNetwork } from '../commons/network.js'
+
+export const addPlugin = (moduleName: Plugin) => {
+  if (Array.isArray(moduleName)) {
+    moduleName.forEach(addPlugin)
+  } else {
+    try {
+      if (!moduleName.startsWith('@perian/')) {
+        throw new Error('Unknown plugin name')
+      }
+      const plugin = moduleName.replace('@perian/', '').split('-')
+      if (plugin[0] === 'network') {
+        // $FlowFixMe
+        addNetwork(plugin[1], require(moduleName))
```

Comment:
This is impossible to package. Have you even tried this on React Native? Module nsmes need to be static strings.

Reply:
Please do what rollup does where the user imports whatever they wants and passes them to Perian as JS objects. This avoids all packaging issues, and is simpler. You can tell if a plugin is a network or whatever by the properties it has.

Reply:
again, this whole thing was removed